lists/transaction-types/README.md

```diff
@@ -0,0 +1,75 @@
+Transaction Type List
+=====================
+
+This is a list of existing, reserved and tentative
+[EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) Transaction Types.
+
+Transaction Types
+-----------------
+
+See the sections below for additional notes on Version Bytes marked
+Reserved or Tentative.
+
+| Version Byte | Specs or Purpose |
+|--------------|------------------|
+| 0x00  | Reserved: indicates legacy (untyped) trancactions |
+| 0x01  | [EIP-2930](https://eips.ethereum.org/EIPS/eip-2930) *(available in Berlin)* |
+| 0x02  | [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) *(available in London)* |
+| 0x03  | Tentative: prevents collision with [EIP-3074](https://eips.ethereum.org/EIPS/eip-3074) |
+| 0x18  | Reserved: prevents collision with [EIP-191](https://eips.ethereum.org/EIPS/eip-191) |
+| 0x80 - 0xff  | Invalid; collides with the initial byte of valid RLP encoded transactions |
+
+
+Reserved Transaction Types Motivation and History
+-------------------------------------------------
+
+Reserved version bytes cannot be used as an [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
+TransactionType and should never be used as a typed transaction prefix.
+
+We disallow prefixing with reserved bytes to prevent encoded transactions from
+colliding with data that may otherwise be signed to prevent unintentionally
+authorizing unintended transactions.
```

Comment:
I believe that for collision avoidance, we *specifically* need to specify the signature type (secp256k1).  If someone signs something with a leading byte of 0x01 but uses a BLS signature (for example) then there is no collision problem I believe?

Reply:
Agreed. I’ll include that. :)

Reply:
I've added this. :)